app/components/DashboardNavigator/ChatScreen/MessagesList/Component.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+import React, { Component } from 'react';
+import { List, FlatList, Text } from 'react-native';
+
+import styles from './Styles';
+
+import MessageRow from './MessageRow';
+import { getChatItems } from '../../../../store/chat/selectors';
```

Comment:
Component should only display data, so move this selector to container and pass them as props